src/test/mocks/store.ts

```diff
@@ -269,6 +269,7 @@ export const mockStore: ReduxState = {
   },
   limitOrderSlice: {
     activeQuote: undefined,
+    orderToCancel: undefined,
```

Comment:
q: do we want this guy to be local-state driven instead, or is there any rationale to having this live in store?
If we do keep it in the store, then state won't reset on swapper unmount meaning we could end up in weird conditions like this one:

https://jam.dev/c/4d258907-f8a1-450f-bdd1-0662420253ad

Reply:
Note, that's most likely fine and also most likely doesn't require a refactor, ensuring `orderToCancel` reset on unmount is probably all we need!

Reply:
Really well noted ser. The redux storage of that was an artefact of a previous iteration of the route implementation which no longer requires sharing state across routes. I've removed it and its local only now.

eff1aef57bbb2bed97cdc751546291c92374ec2d

Reply:
🎉